sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -231,6 +241,32 @@ private [sql] case class CreateTempTableUsing(
   }
 }
 
+private[sql] case class DropTable(
+    tableName: String,
+    isExists: Boolean,
+    temporary: Boolean) extends Command
```

Comment:
Can't we just have one parser handle all drop table commands with a single logical operator.  In the actual planner we can match on the type of the table and delegate to the proper `RunnableCommand`